zebra-scan/src/storage/db.rs

```diff
@@ -78,7 +78,17 @@ impl Storage {
 
         let new_storage = Self { db };
 
-        // TODO: report the last scanned height here?
+        // Report where we are for each key in the database.
+        let keys = new_storage.sapling_keys_last_heights();
+        for (key_num, (_key, height)) in keys.iter().enumerate() {
+            tracing::info!(
+                "Last scanned height for key number {} is {}, resuming at {}",
+                key_num,
+                height.previous().expect("height is not genesis").as_usize(),
+                height.as_usize(),
```

Comment:
We start at the next height after the last scanned height:
```suggestion
                height.as_usize(),
                height.next().expect("height is not maximum").as_usize(),
```